lib/bacalhau.js

```diff
@@ -129,19 +119,18 @@ async function start ({
   } catch (err) {
     const errorMsg = err instanceof Error ? err.message : '' + err
     const message = `Cannot start Bacalhau: ${errorMsg}`
-    logStream.write(message)
-    activityStream.write({ type: 'error', message })
+    activities.emit('activity', { type: 'error', message })
```

Comment:
In the old version, activities were recorded in the logs in addition to being submitted to the activity stream. In the new version, they are reported in one place only. Could you please confirm that by emitting an activity this way, a log line will be printed to Core's stdout?